src/rt/lifetime.d

```diff
@@ -2590,7 +2578,6 @@ unittest
     delete arr1;
     assert(dtorCount == 7);
 
-    if (callStructDtorsDuringGC)
```

Comment:
Please remove scoping/indentation, too. Later it will just confuse people why it is there sometimes, and sometimes not.